Make.inc

```diff
@@ -1412,6 +1412,8 @@ endif
 define dir_target
 $$(abspath $(1)):
 	@mkdir -p $$@
+$$(abspath $(1))/: | $$(abspath $(1))
+	@true
```

Comment:
Don't merge yet. apparently the old version of make on mac used to understand that these were the same path, but the updated versions used everywhere else aren't able to handle this anymore. I guess we need to add this trailing `/` everywhere instead of making it convenient with this added rule